qfieldsync/gui/project_configuration_widget.py

```diff
@@ -193,6 +202,14 @@ def apply(self):
             )
         except AttributeError:
             pass
+        try:
+            self.__project_configuration.digitizing_logs_layer = (
+                self.digitizingLogsLayerComboBox.currentLayer().id()
+                if self.digitizingLogsLayerComboBox.currentLayer()
+                else ""
+            )
+        except AttributeError:
+            pass
```

Comment:
When does it throw?

Reply:
@suricactus , I took the same approach as the base map layer setting, assume it was there for a reason.

Reply:
I also had no idea for the above setting, hoped you found a situation when it throws. The rest of the settings are not surrounded by try/catch. Let's hope for the best. :orthodox_cross: Anyway, it is safer this way. Thanks.